elements/rh-cta/rh-cta.css

```diff
@@ -9,7 +9,7 @@
 ::slotted(*) {
   vertical-align: middle;
   white-space: break-spaces;
-  display: inline !important;
+  display: inline-flex !important;
```

Comment:
For arrows and icons this breaks their inline placement.

Reply:
please take another look

Reply:
@bennypowers is the pf-icon for video supposed to wrap with the text like the arrow does? I can't remember.

Reply:
I took another look at the design docs. Seems to me that the video icon is a special case. Ordinarily, you're only allowed icons in the following cases:

- default variant *must* have *only* the prescribed arrow icon *or* the play icon if it's a video cta
- brick variant *may* have *any* approved icon *except the video icon*, but it appears before the slot text only
- primary or secondary *may* have *only* the video icon if it's the video cta, appearing *after* the link.
- brick variant *may not* have the video icon

Is that correct, @coreyvickery ?

Reply:
If necessary, we'll spin off a new issue for this down the line